test/test_prototype_models.py

```diff
@@ -83,6 +83,38 @@ def test_naming_conventions(model_fn):
     assert len(weights_enum) == 0 or hasattr(weights_enum, "default")
 
 
+@pytest.mark.parametrize(
+    "model_fn",
+    TM.get_models_from_module(models)
+    + TM.get_models_from_module(models.detection)
+    + TM.get_models_from_module(models.quantization)
+    + TM.get_models_from_module(models.segmentation)
+    + TM.get_models_from_module(models.video),
+)
+def test_schema_meta_validation(model_fn):
+    classification_fields = ["size", "categories", "acc@1", "acc@5"]
+    defaults = {
+        "all": ["interpolation", "recipe"],
+        "models": classification_fields,
+        "detection": ["categories"],
```

Comment:
`map` not included here because keypoint and maskrcnn have several differently named mAPs. One option is to rename their `box_map` to `map`.

Reply:
I ended up renaming `box_map` => `map` and added the field as mandatory for detection models.